primitives/consensus/common/src/import_queue/basic_queue.rs

```diff
@@ -117,10 +117,10 @@ impl<B: BlockT, Transaction: Send> ImportQueue<B> for BasicQueue<B, Transaction>
 		who: Origin,
 		hash: B::Hash,
 		number: NumberFor<B>,
-		justification: Justification,
+		justifications: Justifications,
```

Comment:
This should be `Justification`. This will get called as part of the `JustificationImport` trait which will be implemented by the different consensus engines, so each consensus engine will only be able to import its own justifications.